apps/omg_api/lib/block_queue.ex

```diff
@@ -118,6 +119,14 @@ defmodule OMG.API.BlockQueue do
 
       interval = Application.fetch_env!(:omg_api, :block_queue_eth_height_check_interval_ms)
       {:ok, _} = :timer.send_interval(interval, self(), :check_ethereum_status)
+      parent = self()
+
+      {:ok, _} =
```

Comment:
I'm hesitant, but maybe we could DRY this invocation to a one liner like `OMG.API.Recorder.start[_link](name: only_the_name_varies)`

Reply:
Yeah. That might be better. I presume we would want to track other things on a per process basis.